tools/github_workflows/build_and_upload_api_docs.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+# This script is used to build API documentations 
+# and upload the documentation to self-hosted repos.
```

Comment:
This script looks neat! Did you use ChatGPT to clean up the script automatically?

Reply:
Yes, the script was revised by ChatGPT.